src/cljs/main/broadfcui/page/method_repo/method/wdl.cljs

```diff
@@ -8,7 +8,7 @@
 (react/defc WDLViewer
   {:render
    (fn [{:keys [props]}]
-     [:div {:style {:margin "2.5rem 1.5rem"}}
+     [:div {:style {:marginTop "2.5rem" :marginLeft "1.5rem" :marginRight "1.5rem" :marginBottom "1.0rem"}}
```

Comment:
just `1 rem`

Reply:
`{:margin "1.0rem"}`? That does change the horizontal position of CodeMirror on the page (in a way that looks better to me)

Reply:
(Though the URL label is no longer aligned)

Reply:
Oooooh, that was a woosh on my part. Thanks